legend-engine-language-pure-grammar/src/test/java/org/finos/legend/engine/language/pure/grammar/test/TestGrammarRoundtrip.java

```diff
@@ -73,7 +74,9 @@ private static void test(String code, String message)
                 throw new RuntimeException(e);
             }
             PureGrammarComposer grammarTransformer = PureGrammarComposer.newInstance(PureGrammarComposerContext.Builder.newInstance().build());
-            Assert.assertEquals(message, code, grammarTransformer.renderPureModelContextData(modelData));
+            Assert.assertEquals(message,
```

Comment:
why delete whitespace, the grammar generation should be same as what is parsed, and should be  correctly indented. By deleting whitespace during test, we run the risk of poorly formatted grammar generation passing tests.

Reply:
You right. Missed that. Reverted.